locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -3004,6 +3006,14 @@ protected static double toDouble(Object v) {
         }
     }
 
+    protected static int[] toIntPortArray(Object v) {
+        Collection<String> inboundPortEntries = toListOfStrings(v);
```

Comment:
Not fully backwards compatible. `toIntArray` supports the format `[1,2,3]`. Also accepts int arrays if passed from Java code. The first one should definitely be supported, no hard opinion on the second one.
Would be nice to have a test for the new method.

Otherwise looks very useful.